modules/social_features/social_user/src/Plugin/Action/SocialSendEmail.php

```diff
@@ -0,0 +1,198 @@
+<?php
+
+namespace Drupal\social_user\Plugin\Action;
+
+use Drupal\views_bulk_operations\Action\ViewsBulkOperationsActionBase;
+use Drupal\views_bulk_operations\Action\ViewsBulkOperationsPreconfigurationInterface;
+use Drupal\Core\Plugin\PluginFormInterface;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Session\AccountInterface;
+use Drupal\Core\Entity\EntityManagerInterface;
+use Drupal\Core\Language\LanguageManagerInterface;
+use Drupal\Core\Mail\MailManagerInterface;
+use Drupal\Core\Utility\Token;
+use Psr\Log\LoggerInterface;
+use Egulias\EmailValidator\EmailValidator;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * An example action covering most of the possible options.
+ *
+ * @Action(
+ *   id = "social_user_send_email",
+ *   label = @Translation("Send email"),
+ *   type = "user",
+ *   confirm = TRUE,
+ * )
+ */
+class SocialSendEmail extends ViewsBulkOperationsActionBase implements ViewsBulkOperationsPreconfigurationInterface, PluginFormInterface {
```

Comment:
@jaapjan delete **PluginFormInterface** because **buildPreConfigurationForm** is empty.